presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/IntegerStatistics.java

```diff
@@ -57,17 +58,40 @@ public Long getMin()
         return hasMinimum ? minimum : null;
     }
 
+    public long getMinPrimitive()
```

Comment:
why not remove the method getMin completely ?

Reply:
That would be nice, but it comes from the RangeStatistics<Long> interface implemented by IntegerStatistics.

```
public interface RangeStatistics<T>
{
    T getMin();

    T getMax();

    long getRetainedSizeInBytes();
}

```